src/celeritas/phys/ImportedModelAdapter.cc

```diff
@@ -78,17 +78,19 @@ auto ImportedModelAdapter::micro_xs(Applicability applic) const
     auto xs = import_process.micro_xs.find(model_class_);
     CELER_ASSERT(xs != import_process.micro_xs.end());
     CELER_ASSERT(applic.material < xs->second.size());
-    const auto& el_to_vec = xs->second[applic.material.get()];
+    const auto& elem_phys_vectors = xs->second[applic.material.get()];
 
     MicroXsBuilders builders;
-    for (const auto& kv : el_to_vec)
+
+    for (size_type i : range(elem_phys_vectors.size()))
     {
-        const auto& vec = kv.second;
+        const auto& vec = elem_phys_vectors.at(i);
```

Comment:
It's fine for front-end type code, but generally it's better to avoid using `.at()` because it does always-on range checking (and we tend to have our own assertions that we can disable for performance).